mongodb/common/src/main/java/org/apache/metamodel/mongodb/common/MongoDBUtils.java

```diff
@@ -73,5 +101,29 @@ private static Object toValue(Column column, Object value) {
         }
         return value;
     }
+    
+    public static Object covertArrayToList(Object arr) {
```

Comment:
Could we make this method private and just call it in the toValue(...) method instead? Since the method is really restricted in it's use it seems maybe we can add an ```if``` above and then not worry about other people misunderstanding what this is used for.

Small note on naming - it's misspelled (missing "n" in convert).